### ISSUE TITLE
new schema specification from AWS 

### ISSUE THREAD

**Issue by sdole** [shalupov/idea-cloudformation#74]
AWS has published a schema specification for cloudformation. I wonder if this makes it easier to update features on this plugin. 

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html
https://aws.amazon.com/about-aws/whats-new/2016/11/aws-cloudformation-introduces-resource-specifications-cross-stack-reference-update-and-coverage-updates/

**Comment by shalupov** [comment]
It's ok to parse web documentation, but downloading a spec brings support for new resource types without new plugin version as suggested in #76

**Comment by sdole** [comment]
Yeah, the spec urls are on this page: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-resource-specification.html

**Comment by sdole** [comment]
For example, in us-east-1 the url is https://dnwj8swjjbsbt.cloudfront.net/latest/CloudFormationResourceSpecification.json
